website/src/pages/influenza-a/index.astro

```diff
@@ -1,6 +1,9 @@
 ---
-import GenericOverview from '../../components/views/overview/GenericOverview.astro';
+import LinkedOverview from '../../components/views/overview/LinkedOverview.astro';
+import { getPathogenMegaMenuSections } from '../../layouts/base/header/getPathogenMegaMenuSections';
 import { Organisms } from '../../types/Organism';
+
+const subtypes: (keyof ReturnType<typeof getPathogenMegaMenuSections>)[] = ['h5n1', 'h3n2', 'h1n1pdm'];
```

Comment:
Let's please use the constants from `Organisms`. (Also I think you won't need explicit type annotations then, because TS will recognize them as const values. Nitpick: IMO we could also inline `subtypes`).